landoapi/hooks.py

```diff
@@ -30,7 +30,10 @@ def check_maintenance() -> Optional[Response]:
         "dockerflow.version",
         "dockerflow.lbheartbeat",
     )
-    if request.endpoint in excepted_endpoints:
+    treestatus_api_prefix = "landoapi_api_treestatus"
+    if request.endpoint in excepted_endpoints or request.endpoint.startswith(
+        treestatus_api_prefix
+    ):
```

Comment:
nit: I think it would be better to be explicit about all the endpoints that we are excluding here. This would help with debugging.

Reply:
This would basically log all requests to those endpoints all the time. I meant basically listing out all the individual endpoints (similar to how we don't do `request.endpoint.startswith("dockerflow.")`).

This will add unnecessary logs, since the dockerflow endpoints are checked regularly.